Source/objects.h

```diff
@@ -49,6 +50,36 @@ struct ObjectStruct {
 	uint32_t _oVar6;
 	_speech_id _oVar7;
 	int _oVar8;
+
+	/**
+	 * @brief Marks the map region to be refreshed when the player interacts with the object.
+	 *
+	 * Some objects will cause a map region to change when a player interacts with them (e.g. Skeleton King
+	 * antechamber levers). The coordinates used for this region are based on a 40*40 grid overlaying the central
+	 * 80*80 region of the dungeon.
+	 *
+	 * @param topLeft corner of the map region closest to the origin
+	 * @param bottomRight corner of the map region furthest from the origin
+	 * @param v ID/discriminator for the object type? Needs to be investigated further
+	 */
+	constexpr void SetMapRange(Point topLeft, Point bottomRight, int v)
+	{
+		_oVar1 = topLeft.x;
+		_oVar2 = topLeft.y;
+		_oVar3 = bottomRight.x;
+		_oVar4 = bottomRight.y;

```

Comment:
Are `oVar`s used for other things? Asking just in case it would be possible to unify them into a single `Rectangle` variable.

Reply:
Unfortunately yes. They're general purpose variables for all sorts of object types and the ones at the start tend to be the most commonly used.